data-table/table.js

```diff
@@ -58,22 +200,37 @@ function onColumnEntryBoxEnter(event, boxValue) {
      */
 }
 
-function addColumn(numberOfColumns) {
+function addColumns(numberOfColumns, index) {
     /**
      * Adds columns from existing table. Will probably call
      * createEntryCell(). Maybe should also accept index as an argument?
      * TODO: Fill this out
      */
 }
 
-function deleteColumn(numberOfColumns) {
+function addSingleColumn(index) {
+    /**
+     * Adds single column from existing table. Will probably call
+     * createEntryCell(). Maybe should also accept index as an argument?
+     * TODO: Fill this out
+     */
+}
```

Comment:
For simplicity (for now), consider simplifying this and others to never take an index: always add at the end, and delete from the end. Later, we can go back and add functionality to add and delete arbitrary rows, but that becomes difficult because you may have to rename the IDs of all others cells.

Reply:
I *think* I'm gonna keep the index for now: it's being used during table generation to create the unique, index-specific HTML IDs for each cell